packages/grafana-ui/src/components/Logs/LogRows.tsx

```diff
@@ -28,6 +28,7 @@ export interface Props extends Themeable2 {
   previewLimit?: number;
   forceEscape?: boolean;
   showDetectedFields?: string[];
+  scrollElement?: HTMLDivElement | undefined;
```

Comment:
```suggestion
  scrollElement?: HTMLDivElement;
```

If it is already optional, can't we omit `undefined` here?

Reply:
Uff, good catch. `scrollElement` on the "old" `LogRows` in grafana-ui is not foreseen. I removed it here and also removed `undefined` from the other components in https://github.com/grafana/grafana/pull/55989/commits/ec38566f2c2e48d6da9830e01f786e333a22160d